vwifi.c

```diff
@@ -1611,7 +1613,41 @@ static int vwifi_change_beacon(struct wiphy *wiphy,
 
     return 0;
 }
+#else
```

Comment:
In my opinion, surrounding the changed lines with the `#if` directives is better since there are only a few lines changed.

That is:
```c
static int vwifi_change_beacon(struct wiphy *wiphy,
                               struct net_device *ndev,
#if LINUX_VERSION_CODE < KERNEL_VERSION(6, 7, 0)
                               struct cfg80211_beacon_data *info
#else
                               struct cfg80211_ap_update *info)
#endif
```

Reply:
However, due to the changes within this function, the frequency of using `#if` will significantly increase.